pkg/daemon/vm_controller.go

```diff
@@ -164,6 +164,19 @@ func (r *VMReconciler) reconcile(ctx context.Context, vm *virtv1alpha1.VirtualMa
 			switch vm.Status.Migration.Phase {
 			case virtv1alpha1.VirtualMachineMigrationScheduled:
 				if vm.Status.Migration.TargetNodeName == r.NodeName {
+					if err := wait.PollImmediate(time.Second, 3*time.Second, func() (done bool, err error) {
```

Comment:
Is this change related with this issue?

Reply:
yes, now in virt-prerunner, cloud-hypervisor is running by s6 as a service, when the pod become `running`, cloud-hypervisor maybe maybe is not running and the socket file is not created. so we have to wait it.